packages/application-shell/src/components/fetch-project/fetch-project.spec.js

```diff
@@ -12,18 +12,7 @@ import FetchProject from './fetch-project';
 
 jest.mock('@commercetools-frontend/sentry');
 
-const mockServer = setupServer(
-  graphql.query('FetchProject', (req, res, ctx) =>
```

Comment:
The idea of having this here was that in the tests we don't need to provide any mocks.

Reply:
Okay, what's the change request being made? Can you commit it maybe if you've something in mind?